### ISSUE TITLE
Update of core

### ISSUE THREAD

**Issue by Papermanzero** [libretro/dolphin#159]
The main branch received several updates. Especially the new lossless format RVZ is a great achievement.

**Comment by ordovice** [comment]
+1 for an upstream update

**Comment by i30817** [comment]
You may have had your wish:

https://github.com/libretro/dolphin/commit/56e9a017d0dffe923391ea750f9aa914486a1c45

**Comment by Papermanzero** [comment]
Thanks a lot!!!

**Comment by Mr-Jake** [comment]
It does look like it was updated, but not built yet on the buildbot. I'm hoping the new RVZ compressed image format is supported by the core. 

Big thanks to the Libretro team for updating the Dolphin core!

**Comment by i30817** [comment]
I don't think it does support RVZ. Or at least a search for 'RVZ' on the repository doesn't show any commits or code for the string. Which is kind of weird if you ask me, if dolphin master was merged.

ie compare these searches:

dolphin: https://github.com/dolphin-emu/dolphin/search?q=RVZ_MAGIC&unscoped_q=RVZ_MAGIC

libretro dolphin: https://github.com/libretro/dolphin/search?q=RVZ_MAGIC&unscoped_q=RVZ_MAGIC


Maybe github hasn't indexed the code yet, lets see if the files exist.

edit: yes, it was that, the RVZ strings are in those files in libretro. Whew.

**Comment by Papermanzero** [comment]
It supports RVZ with the newest updates.

**Comment by Formedras** [comment]
Source/Core/DolphinLibretro/Main.cpp still needs `void retro_get_system_info(retro_system_info* info)` to be updated for the WIA and RVZ file extensions. I tried downloading and compliling myself, but something went wrong, and I don't wanna try submitting a pull request unless I can do a successful build.

EDIT: Oh also I don't have permissions to do Pull Requests. Which I found out after getting a successful build. Or at least I don't think I do. Development is confusing sometimes.
EDIT 2: Ok, forked, committed to my repo, Pull Request made. Yay I'm helping. I think. #179

**Comment by Papermanzero** [comment]
https://github.com/hunterk/libretro_builds

those builds run rvz without issue

**Comment by Formedras** [comment]
Well yes they **run** RVZ images without issue, but my point wasn't about running them, it was about detecting them in the RetroArch GUI. Although the core could handle the files just fine if given an RVZ or WIA file, it wasn't telling RetroArch (or other Libretro frontends) that they're acceptable, so they wouldn't show up in the file browser.

**Comment by Papermanzero** [comment]
I see. Well, I am generating my playlists manually due to certain rom hacks and translations. But awesome that you implemented that feature 👍

**Comment by ThisNekoGuy** [comment]
Adding RVZ detection to the GUI frontend, officially, would be helpful :p

**Comment by gouchi** [comment]
Hi @Papermanzero,

The core has been updated recently with this https://github.com/libretro/dolphin/pull/352 thanks to [cscd98](https://github.com/cscd98) !

So please close this issue.

Thank you.
